js/layout-javascript/news-feed-code.js

```diff
@@ -1643,7 +1643,7 @@ DynamicList.prototype.searchData = function(options) {
 
       if (limitEntriesEnabled) {
         // Do not show limit text when user is searching or filtering
-        _this.$container.find('.limit-entries-text').toggleClass('hidden', truncated);
+        _this.$container.find('.limit-entries-text').toggleClass('hidden', results.filteredResult);
```

Comment:
Shouldn't `truncated` be used here in some way as well? Otherwise the message would be shown when there are less entries than the limit in the initial render.

```suggestion
        _this.$container.find('.limit-entries-text').toggleClass('hidden', !truncated || results.filteredResult);
```

Reply:
@tonytlwu 
Actually it works fine without truncated variable. During testing found another bug, made commit to fix it. Should work correctly now.

Reply:
@MaksymShokin Can you explain why it isn't needed?

Reply:
@tonytlwu 
I tested it and it works without truncated.

Reply:
@MaksymShokin Yes, that sounds like what you said previously. I needed an explanation for why it isn't needed, not the results. It needs to be explained because before your changes in https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/363/files `truncated` was part of the logic. Because you're now effectively removing it from the logic, it needs to be explained when asked.

Here's an example.

I took a look at your changes, tested it myself and looked into the code. I found that [this code](https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/a3bb49bcf81a1ba70aac0c759e32f497cc595f8f/js/layout-javascript/news-feed-code.js#L1552-L1555) suggests that `limit` is only a valid number (above `-1`) if search, filter and bookmarks are disabled.

This suggests that `results.truncated` will never be `true` if search/filter/bookmarks are enabled. It also means that the solution from the beginning could just be:

```js
_this.$container.find('.limit-entries-text')[truncated && limit > 0 ? 'removeClass' : 'addClass']('hidden');
```

This could work because the variable `limit` has already taken the search, filter and bookmark states into consideration. It's a simpler solution, requires no new logic wither in the layout JS or the `utils.js` files.

What do you think? Would this work, and do you have any reasons for preferring other solutions?